pallets/loans-ref/src/types.rs

```diff
@@ -622,6 +676,41 @@ impl<T: Config> ActiveLoan<T> {
 		Ok(())
 	}
 
+	fn ensure_can_modify(&self, mutation: &Mutation<T::Rate>) -> DispatchResult {
+		// What happens if it has overdue? or has been repaid or written off?
```

Comment:
Suppose the following:

I want to modify the interest rate.
1. I propose the change X
2. While time passes, it is repaid/reborrowed/written-off,
3. The change is ready to be applied.

Depending on change X or any amount of actions performed during that time, the answer of "what to do when the change is applied" could be different.

Reply:
Given the current changes, I am not sure this is a problem.

Reply:
The only thing I am not sure is interest rate if the rate has been adated through write-off. We could check that right? If the write-off-status of a loan is set

Reply:
Yes, in fact, I think currently I'm not doing it, and I should.

Reply:
Done!